cmake/onnxruntime_java.cmake

```diff
@@ -171,8 +171,8 @@ if (CMAKE_SYSTEM_NAME STREQUAL "Android")
   file(MAKE_DIRECTORY ${ANDROID_PACKAGE_ABI_DIR})
 
   # Create symbolic links for onnxruntime.so and onnxruntime4j_jni.so for building Android AAR package
```

Comment:
>Create symbolic links [](http://example.com/codeflow?start=4&length=21)

this comment seems to be inaccurate now. why is it changed to copy?

Reply:
If we copy the binaries to from one dir to another dir then run the tests, all symlink will be broken and the tests will fail. 
Also, symbolic link doesn't work well on Windows. It works only for a clean build. If you changed any source file and build it again without delete the whole build dir first, the build will fail.